improvement(custom_d1): use greatly improved latte+rune scripts

vponomaryov opened the pull request:
### Testing
<!-- Add links to Argus/Jenkins of test test done with this PR -->
<!-- This would help the reviewer to cross check what was tested, and and review the results as needed -->
- 3h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#8](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=cb36d00e-6b3d-4342-a709-eb93e9a9c9b8) (PASSED)
- 12h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#10](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=f92f22e5-c0b4-4d80-9ea0-7e2785d749e7) (PASSED)

### PR pre-checks (self review)
<!--- PR should be created as Draft, when CI finished and relevant checkboxes selected, add reviewers and then click on "Ready for review" button.-->
<!--- Put an `x` in all the boxes that apply or create PR and then click on all relevant checkboxes: -->
- [ ] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevent to this change (if needed)

vponomaryov commented:
@fruch it is workable, just intermediate state. Will update it according to our discussion.

fruch commented on test-cases/longevity/longevity-gce-custom-d1-hybrid-raid.yaml:
```diff
@@ -3,24 +3,84 @@ n_db_nodes: 6
+# TODO: enable 'nemesis_during_prepare' back when debug of stress commands get finished
+nemesis_during_prepare: false
 
 gce_instance_type_db: 'n2-highmem-8'
```
we should increase it to n2-highmem-32

vponomaryov replied:
I am working on it.

vponomaryov replied:
Done

fruch reviewed (commented):
Looks good, it would be helpful to see some of the links to argus, to see how it look at the end

vponomaryov commented:
> Looks good, it would be helpful to see some of the links to argus, to see how it look at the end

[scylla-staging/valerii/vp-longevity-d1-hybrid-raid-n2-highmem-32-2022.1#15](https://argus.scylladb.com/test/db271c9f-b07d-4644-8df5-ad368d9f3a5c/runs?additionalRuns[]=f0793996-2a3e-453d-ae84-bd6e83534cd5)

vponomaryov commented:
> Looks good, it would be helpful to see some of the links to argus, to see how it look at the end

- 3h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#8](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=cb36d00e-6b3d-4342-a709-eb93e9a9c9b8) (PASSED)
- 12h: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#10](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=f92f22e5-c0b4-4d80-9ea0-7e2785d749e7) (PASSED)

vponomaryov commented:
It is being tested here: [scylla-staging/valerii/vp-longevity-gce-custom-d1-hybrid-raid-230kops#15](https://argus.scylladb.com/test/1d32954c-742b-4c40-b45d-74d8ada43e78/runs?additionalRuns[]=f01a2dd0-fe64-4eaf-8194-312c143bb057)

So, this is really working solution.
Open item is to pick up some nemesis_seed.
From my point of view, it is ready to be merged.

vponomaryov commented:
@roydahan , @fruch This PR is waiting for you

roydahan commented:
I won't be able to reach it before the week after the summit.

vponomaryov commented:
> @roydahan , @fruch This PR is waiting for you

Still true

roydahan reviewed (approved):
I'm not into the details of the customer profile in order to really review the latte command line parameters but judging from the test results and reviewing everything else, it looks good to me.

vponomaryov commented:
Changes:
- Updated the docker images to contain fix for memory leaks
- Switched the instance type for loaders to cheaper ones - `e2-highcpu-32` instead of the `e2-standard-32`.

fruch commented:
@vponomaryov need to fix those

vponomaryov commented:
> @vponomaryov need to fix those

Done
